ecodev_core/filter_db_filters.py

```diff
@@ -0,0 +1,283 @@
+"""
+Low level methods to retrieve data from db in a paginated way, using a filter model
+"""
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from ecodev_core import engine
+from ecodev_core import logger_get
+from ecodev_core import ServerSideField
+from ecodev_core import ServerSideFilter
+from ecodev_core.db_filters import SERVER_SIDE_FILTERS
+from sqlalchemy import func
+from sqlalchemy.orm import InstrumentedAttribute
+from sqlmodel import col
+from sqlmodel import or_
+from sqlmodel import select
+from sqlmodel import Session
+from sqlmodel.sql.expression import SelectOfScalar
+
+log = logger_get(__name__)
+
+
+def count_rows(
```

Comment:
This module is quite technical, and basically a redo of `db_retrievals` for dash ag grid instead of data tables (which is way better, thanks!).

If possible, you can spend some time on trying to unify what can be, so that we have a high level `count_rows` and `get_rows` with a feature flag (or a two value enum) for using one or the other. I have the feeling (but now you have a fresher view on the topic than me) that not so much is different from the two approaches.